cmd/podman/common/completion.go

```diff
@@ -778,10 +778,10 @@ func AutocompleteImageVolume(cmd *cobra.Command, args []string, toComplete strin
 }
 
 // AutocompleteLogDriver - Autocomplete log-driver options.
-// -> "journald", "none", "k8s-file"
+// -> "journald", "none", "k8s-file", "passthrough"
 func AutocompleteLogDriver(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 	// don't show json-file
-	logDrivers := []string{define.JournaldLogging, define.NoLogging, define.KubernetesLogging}
+	logDrivers := []string{define.JournaldLogging, define.NoLogging, define.KubernetesLogging, define.PassthroughLogging}
```

Comment:
Small nit: Can you use `registry.IsRemote()` and only add passthrough if we are local. I don't want to recommend options to users that cannot be used.

Reply:
sure.  Pushed a new version.